extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -40,13 +40,23 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 		return allFiles;
 	}
 
-	async openNotebook(resource: vscode.Uri): Promise<void> {
+	async openNotebook(resource: string): Promise<void> {
```

Comment:
did we need to change this (and openMarkdown) to take in a string as a parameter? Could we have done something different in the caller? Just a quick explanation would help me 😄

Reply:
I was passing in pathToNotebook and pathToMarkdown, both of which are strings. Uri isn't needed in openNotebook and I convert resource to an Uri in openMarkdown since the showPreview method needs it.